### PR TITLE
[#164416800] Move overlay info to instabug and profile screen

### PR DESCRIPTION
**Pull request by fpersico**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#164416800](https://www.pivotaltracker.com/story/show/164416800): Nella versione per la closed-beta le informazioni di debug su versione backend e route corrente devono essere spostate dallo schermo ad attributi Instabug 

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/boot/configureInstabug.ts [hunk 1]
```diff
@@ -29,6 +29,13 @@ export const initialiseInstabug = () => {
   );
 };
 
+export const setInstabugUserAttribute = (
```
Comment (cloudify) [inline ts/boot/configureInstabug.ts]: uhm what is the added value of this function?

File: ts/config.ts [hunk 2]
```diff
@@ -72,6 +72,9 @@ export const totMessageFetchWorkers = t.Integer.decode(
   Config.TOT_MESSAGE_FETCH_WORKERS
 ).getOrElse(DEFAULT_TOT_MESSAGE_FETCH_WORKERS);
 
+export const displayVersionInfoOverlay =
```
Comment (cloudify) [inline ts/config.ts]:
```suggestion
export const shouldDisplayVersionInfoOverlay =
```

File: ts/boot/configureInstabug.ts [hunk 3]
```diff
@@ -29,6 +29,13 @@ export const initialiseInstabug = () => {
...
+  attributeKey: string,
```
Comment (cloudify) [inline ts/boot/configureInstabug.ts]: to make this really useful you could use it to type check the `attributeKey` with

```suggestion
  attributeKey: "backendVersion" | "activeScreen",
```

File: locales/en/index.yml [hunk 4]
```diff
@@ -11,7 +11,7 @@ global:
     reference: !include markdown_reference.md
     decodeError: Message can't be decoded
   app:
```
Comment (cloudify) [inline locales/en/index.yml]: the location of these two keys doesn't look good, they should both be under "profile" (the keys are organized per screen - the only exception is the keys for generic actions like ok/cancel/etc...)

**Pull request merged**